docs/tutorials/general.md

```diff
@@ -0,0 +1,350 @@
+cd oasgraph
+npm link
+```
+
+Please note that OASGraph can be used either as a library, or via its Command Line Interface (CLI). In this case, we will be using the CLI tool, which will start a server in addition to creating the GraphQL interface.
+
+### Start GraphQL server
+
+Start the GraphQL server by running the following command.
+
+```
+oasgraph [path to saved OAS]
+```
+
+The created GraphQL server is then accessible at [http://127.0.0.1:3001/graphql](http://127.0.0.1:3001/graphql).
+
+### Try simple queries
+
+Congratulations! You have created your first GraphQL interface with OASGraph.
+
+We can explore the GraphQL interface through the [GraphiQL IDE](https://github.com/graphql/graphiql) that will show at the previously provided link.
+
+Try to run a simple query like the following:
+
+GraphQL query
+```
+query{
+  person(id: 1){
+    name
+  }
+}
+```
+
+Expected output
+```
+{
+  "data": {
+    "person": {
+      "name": "Albert"
+    }
+  }
+}
+```
+
+***
+
+Although this is a fully-fledged GraphQL interface, we can still improve it.
+
+For instance, this GraphQL interface does not provide nested queries. Recall that the **Person object** also stores the *motherId* and *fatherId* which can be used to query other **People**. In GraphQL, we can make it so that a user can query the mother and father objects from within the child object. To do so, we will need to make slight additions to the OAS, which will be discussed in the following sections.
+
+For now, in the *Documentation Explorer* on the right hand side, click **Query** and then the return type, **Person**, in ```person(id: Float!): person```. Note that the **Person** object can only return the following fields. With our nested query changes, we will see new fields that will let us query the mother and father objects.
+
+```
+id: Float
+name: String
+generation: Float
+motherId: Float
+fatherId: Float
+```
+
+### Adding links
+
+[Link objects](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject "Link object documentation") are constructs in OAS that allow you to explicitly define relationships between different models and operations. For example, when you query for a **Person**, you will receive a *motherId* and a *fatherId*, which can be used to query for other **People**. When you have actions that naturally lead to the next, you use **Link objects** to describe this flow.
+
+Even though **Link objects** (and OAS in general) are useful for documenting APIs, links are especially important for OASGraph. Even though OASGraph can wrap any API with a valid OAS, it needs **Link objects** in order to create GraphQL interfaces that can conduct the powerful nested queries they are famous for. If we try to use the OAS now, the GraphQL interface OASGraph will create will only be able to handle simple queries that are no different than the ones the original API is capable of. Luckily, **Link objects** are easy to define.
```

Comment:
Maybe replace "Even though..." with "While..."? Otherwise it sounds like you are try to make an opposite point, not an extending one.